packages/admin/admin/src/form/file/FileSelect.tsx

```diff
@@ -47,6 +52,7 @@ export type FileSelectProps<AdditionalValidFileValues = Record<string, unknown>>
     maxFiles?: number;
     multiple?: boolean;
     error?: React.ReactNode;
+    showFilesAsPreviewGrid?: boolean;
```

Comment:
What do you think about using an enum instead? For instance, `layout = "grid"  | "list"`

Reply:
Had something like this previously but didn't like the variable names I chose 😅 
I like your suggestion, I'll do it that way 👍🏼

Reply:
Maybe `variant` instead of `layout`? 
Would be more consistent with other components 🤔

Reply:
> Maybe variant instead of layout?

Wouldn't this conflict with Field's variant? But yes, I'd also go for consistency

Reply:
> Wouldn't this conflict with Field's variant?

Ah, yes it would 😄 
I'll use `layout` then 👍🏼